docker/platypus-deps/Dockerfile

```diff
@@ -113,7 +113,7 @@ RUN export PETSC_DIR=/$WORKDIR/petsc
 WORKDIR /$WORKDIR
 RUN git clone https://github.com/Heinrich-BR/mfem.git
```

Comment:
Not relevant for this PR, but should we switch over to using the main MFEM repo, now the features from our branch have been merged?

Reply:
We should indeed - I'll put a quick PR up today to do this